package.json

```diff
@@ -140,6 +141,11 @@
 	"dependencies": {
 		"array-move": "^3.0.1",
 		"classnames": "^2.2.6",
+		"eslint": "^7.18.0",
```

Comment:
we could move those dependencies to the dev ones since are not present in the final build.